### PR TITLE
Euler method in js

### PR DESCRIPTION
**Pull request by Ariana1729**
+Fixed the ODE(hopefully), cuz there was an inconsistency with ODE and the code

### CODE REVIEW

**Comment by Ariana1729** [comment]
oops uh the bash commit is there

**Comment by Ariana1729** [comment]
Ah ok

**Comment by Ariana1729** [comment]
oops misspelt XD

File: contents/forward_euler_method/code/javascript/euler.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+function solveEuler(timeStep, n){
+  const result = [];
+  if (n != 0) {
+    result[0] = 1;
+    for (let i = 1; i < n; ++i) {
+      result[i] = result[i-1] - 3 * result[i-1] * timeStep;
```
Comment (PudottaPommin) [inline contents/forward_euler_method/code/javascript/euler.js]: `result[i-1]` this should use spaces around `-` so it looks like `result[i - 1]`

**Comment by Ariana1729** [comment]
oh whoops missed that PR

**Pull request closed without merging**